packages/devtools-client-adapters/src/chrome.js

```diff
@@ -1,32 +1,36 @@
+// @flow
+
 const CDP = require("chrome-remote-interface");
 const { getValue } = require("devtools-config");
 const networkRequest = require("devtools-network-request");
 const { setupCommands, clientCommands } = require("./chrome/commands");
 const { setupEvents, clientEvents } = require("./chrome/events");
-const { Tab } = require("./tcomb-types");
-
-// TODO: figure out a way to avoid patching native prototypes.
-// Unfortunately the Chrome client requires it to work.
 
-Array.prototype.peekLast = function() { // eslint-disable-line
-  return this[this.length - 1];
-};
+// similar to Tab, but has webSocketDebuggerUrl
+type ChromeTab = {
```

Comment:
I'd like to extend `Tab` with `webSocketDebuggerUrl`

but i don't think that will work

```
type ChromeTab = {...Tab, webSocketDebuggerUrl: string }
```